src/frecuencies.ts

```diff
@@ -0,0 +1,16 @@
+import { IFrecuency } from './interfaces'
+
+export const FRECUENCIES: IFrecuency[] = [
+  {
+    name: 'Month',
+    code: 'month',
+  },
+  {
+    name: 'Week',
+    code: 'week',
+  },
+  {
+    name: 'Live streaming',
+    code: 'livestreaming',
```

Comment:
just straming is ok, the concept of live streaming is from video actually, in moneay is just streaming
```suggestion
    code: 'streaming',
```